### PR TITLE
Disable form elements when awaiting a server response.

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

**Comment by jankeromnes** [comment]
This partly addresses #128 by disabling form buttons and input elements when you submit asynchronously. Everything gets re-enabled when the form status changes (i.e. success or error).

Some users took the habit of clicking on our buttons multiple times. This change addresses this as well.

**Comment by jankeromnes** [comment]
@nt1m Please have a look.

File: static/js/janitor.js [hunk 1]
```diff
@@ -155,8 +155,16 @@ function setupAsyncForm (form) {
     addFormFeedback(form);
   }
 
-  // Ensure that submitting the <form> doesn't reload the page.
   form.addEventListener('submit', function (event) {
+    // Disable all form elements while waiting for the server.
+    Array.map(form.elements, function (element) {
```
Comment (nt1m) [inline static/js/janitor.js]: nit: use forEach, not map.

Comment (nt1m) [inline static/js/janitor.js]: Also, while you're changing this, can you update https://github.com/jankeromnes/janitor/blob/cc7bf393eff4dde18697b84cb85f6839ec1af0d9/static/js/janitor.js#L183 as well ?

Comment (jankeromnes) [inline static/js/janitor.js]: Done! 👍

**Pull request merged**